inference-engine/thirdparty/clDNN/api/device.hpp

```diff
@@ -55,6 +55,8 @@ struct device_info {
 
     std::string dev_name;     ///< Device ID string
     std::string driver_version;  ///< Version of OpenCL driver
+
+    bool is_discrete;  ///< Whether current device is discrete or integrated GPU
```

Comment:
Can we just reuse OpenCL terminology instead of adding new one?

`CL_DEVICE_HOST_UNIFIED_MEMORY` => `is_host_unified_memory`

/cc @AdamCetnerowski

Reply:
To my mind discrete/integrated terminology is more widely used. Moreover, CL_DEVICE_HOST_UNIFIED_MEMORY query is deprecated in 2.x OpenCL, so I don't think that we need to align our plugin with deprecated thing.